pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java

```diff
@@ -1204,4 +1307,12 @@ public ByteBuffer process(ByteBuffer input, Context context) throws Exception {
             return input;
         }
     }
+
+    public static class SpendTimeFunction implements org.apache.pulsar.functions.api.Function<String, String> {
```

Comment:
`org.apache.pulsar.functions.api.examples.JavaNativeAsyncExclamationFunction` can be instead of this function, please remove this class.

BTW, this is a sync function, not async.